atecc508a_config_dev.h

```diff
@@ -0,0 +1,182 @@
+   * (8-11)  0000   WriteKey - Irrelevant, since clear writes are permitted.
+   * (7)     1      IsSecret - The contents of this slot are secret.
+   * (6)     0      EncryptRead - Reads from this slot are prohibited.
+   * (5)     0      LimitedUse - Unlimited use.
+   * (4)     0      NoMac - The key stored in the slot can be used by
+   *                        all commands.
+   * (0-3)   0111   ReadKey - (priv key in slot) - External and internal
+   *                          signatures are enabled, ECDH operations too. */
+  0x87, 0x20, /* 20-21 SlotConfig 0 */
+  0x87, 0x20, /* 22-23 SlotConfig 1 */
+  0x87, 0x20, /* 24-25 SlotConfig 2 */
+  0x87, 0x20, /* 26-27 SlotConfig 3 */
+  0x87, 0x20, /* 28-29 SlotConfig 4 */
+  0x87, 0x20, /* 30-31 SlotConfig 5 */
+  /* Slot 6 has a different ECDH operation mode - instead of outputing
+   * ECDH master secret in the clear - it uses slot n+1 (7) to write it to.
+   * 0x8F20, ReadKey is 1111. */
+  0x8F, 0x20, /* 32-33 SlotConfig 6 */
+  0x87, 0x20, /* 34-35 SlotConfig 7 */
+
+  /* Slot 8: data storage, slots 9-14: public keys - 0x0000
+   * (12-15) 0000   WriteConfig - Clear text write permitted.
+   * (8-11)  0000   WriteKey - Irrelevant, since clear writes are permitted.
+   * (7)     0      IsSecret - Enabled clear text reads.
+   * (6)     0      EncryptRead - Enabled clear text reads.
+   * (5)     0      LimitedUse - Unlimited use.
+   * (4)     0      NoMac - The key stored in the slot can be used by all commands.
+   * (0-3)   0000   ReadKey - This slot can be the source for the CheckMac/Copy operation. */
+  0x00, 0x00, /* 36-37 SlotConfig 8 */
+  0x00, 0x00, /* 38-39 SlotConfig 9 */
+  0x00, 0x00, /* 40-41 SlotConfig 10 */
+  0x00, 0x00, /* 42-43 SlotConfig 11 */
+  0x00, 0x00, /* 44-45 SlotConfig 12 */
+  0x00, 0x00, /* 46-47 SlotConfig 13 */
+  0x00, 0x00, /* 48-49 SlotConfig 14 */
+/* Slot 15: limited use private key.
+   * 0xA720 = 1010 0111 0010 0000
+   * (12-15) 0010   WriteConfig - Clear text write permitted, GenKey may be
+   *                              used to write random keys into this slot.
+   * (8-11)  0000   WriteKey - Irrelevant, since clear writes are permitted.
+   * (7)     1      IsSecret - The contents of this slot are secret.
+   * (6)     0      EncryptRead - Reads from this slot are prohibited.
+   * (5)     1      LimitedUse - Limited use according to the unique
+   *                             slot 15 rules.
+   * (4)     0      NoMac - The key stored in the slot can be used by
+   *                        all commands.
+   * (0-3)   0111   ReadKey - (priv key in slot) - External and internal
+   *                          signatures are enabled, ECDH operations too. */
+  0xA7, 0x20, /* 50-51 SlotConfig 15 */
+
+  /* Monotonic counter that can be connected to keys to determine how many
+   * times a key can be used. 0x00 means no limit. */
+  0xFF, 0xFF, 0xFF, 0xFF, 0x00, 0x00, 0x00, 0x00, /* 52-59 Counter[0] */
```

Comment:
This is the default value of the monotonic counter, being half of the available range. The counter counts up until it hits  2,097,151. If they were all 0x00, unlimited use would be set and the counter would not count up in any case. In case of this configuration, since the counter is not attached to any of the keys, this value is irrelevant, and hence - left as the default one.